src/meta/src/controller/catalog.rs

```diff
@@ -630,6 +630,7 @@ impl CatalogController {
     }
 
     /// `clean_dirty_creating_jobs` cleans up creating jobs that are creating in Foreground mode or in Initial status.
+    /// FIXME(kwannoel): Notify deleted objects to the frontend.
```

Comment:
Not sure if this can be critical to avoid introducing confusion to users. 🤔

Reply:
That's done in v2 PR.